custom/icds_reports/migrations/sql_templates/database_views/bihar_api_mother_view.sql

```diff
@@ -19,7 +19,8 @@ CREATE VIEW bihar_api_mother_view AS
         "ccs_record_monthly"."tt_1" AS "tt_1",
         "ccs_record_monthly"."tt_2" AS "tt_2",
         "ccs_record_monthly"."anemia" AS "hb",
-        "ccs_record_monthly"."add" AS "add"
+        "ccs_record_monthly"."add" AS "add",
+        "ccs_record_monthly"."edd" - 280 AS lmp
```

Comment:
what's the datatype of this column? I'm surprised this didn't need an `interval` or something, but I assume it works...

Reply:
hey, column is of date type,  yes it works.